proto/streamlit/proto/PlotlyChart.proto

```diff
@@ -27,8 +27,8 @@ message PlotlyChart {
     Figure figure = 2;
   }
 
-  int32 width = 3;
-  int32 height = 4;
+  // If True, will overwrite the chart width spec to fit to container.
+  bool use_container_width = 3;
```

Comment:
You can't re-use `3` since it was being used by `width` before. 

Set this to `5`

Also add a comment at the bottom with 

`// deprecated: 3, 4`

Reply:
For reference: https://github.com/streamlit/streamlit/pull/717#discussion_r352258043